api/v1alpha1/machineconfig_types.go

```diff
@@ -0,0 +1,80 @@
+/*
+Copyright 2022.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
+// NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
+
+// MachineconfigSpec defines the desired state of Machineconfig
+type MachineconfigSpec struct {
+	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
+	// Important: Run "make" to regenerate code after modifying this file
+
+	// EnableCrioProfiling is for enabling profiling of CRI-O service
+	EnableCrioProfiling bool `json:"enableCrioProfiling,omitempty"`
+
+	// EnableKubeletProfiling is for enabling profiling of CRI-O service
+	EnableKubeletProfiling bool `json:"enableKubeletProfiling,omitempty"`
+}
+
+// MachineconfigStatus defines the observed state of Machineconfig
+type MachineconfigStatus struct {
+	// INSERT ADDITIONAL STATUS FIELD - define observed state of cluster
+	// Important: Run "make" to regenerate code after modifying this file
+
+	// LastUpdate is the timestamp of previous status update
+	LastUpdate *metav1.Time `json:"lastUpdated,omitempty"`
```

Comment:
`LastUpdate` or `lastUpdated`?

Reply:
comment incorporated in [d0805fd](https://github.com/openshift/node-observability-operator/pull/16/commits/d0805fd9c9a88344d99aa95ea52cf86ac8ad105c)